### PR TITLE
Fix manual build instructions

### PR DESCRIPTION
**Pull request by voronkovich**
It's needed to run `composer install` (to install the `box`) before calling `make build`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,8 +52,9 @@ $ composer global require manala/manalize
 
 #### Using git:
 ```
-$ git clone git@github.com:manala/manalize
+$ git clone https://github.com/manala/manalize
```
Comment (ogizanagi) [inline README.md]: Why this change?

Comment (voronkovich) [reply README.md]: @ogizanagi, Because git asks a password when `git@github.com:manala/manalize` is used.

Comment (ogizanagi) [reply README.md]: You probably miss your ssh key then, because `git@github.com:manala/manalize` is using SSH.

Using HTTPS, it also asks for your credentials, but you probably did enter them only once if you use a credential helper.

See: 
- https://help.github.com/articles/why-is-git-always-asking-for-my-password/
- https://help.github.com/articles/caching-your-github-password-in-git/

For what it's worth, I'd still recommend using SSH, however it seems [Github recommends using HTTPS](https://help.github.com/articles/which-remote-url-should-i-use/), even if [they changed many times their recommendations](https://stackoverflow.com/a/11041782)...

A good sum up of the different ways to access github repos: https://gist.github.com/grawity/4392747

So, why not.

Comment (voronkovich) [reply README.md]: @ogizanagi, Thanks for the explanation! I've reverted the `git` protocol!

**Comment by ogizanagi** [comment]
Good catch, thanks @voronkovich.
